x/profile/client/cli/tx_update_validator_description.go

```diff
@@ -0,0 +1,70 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/profile/types"
+)
+
+const (
+	flagMoniker         = "moniker"
+	flagSecurityContact = "securityContact"
```

Comment:
```suggestion
	flagSecurityContact = "security-contact"
```

kebab case for flags